src/components/Header/SortResultsSection/SortResultsSection.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SortResultsSection from './SortResultsSection';
+
```

Comment:
Let's test specific things for these components instead of just creating snapshots.